frontend/src/lib/components/PropertyFilters/components/TaxonomicPropertyFilter.tsx

```diff
@@ -81,7 +81,7 @@ export function TaxonomicPropertyFilter({
         eventNames,
         propertyAllowList,
     })
-    const { filter, dropdownOpen, selectedCohortName, activeTaxonomicGroup } = useValues(logic)
```

Comment:
don't use this selector any more; by the time we reach this bit of code we'll always have a `cohort_name` on the filter.